frontendActions/src/main/java/com/virtuslab/gitmachete/frontend/actions/BaseRebaseBranchOntoParentAction.java

```diff
@@ -94,31 +89,35 @@ public void update(AnActionEvent anActionEvent) {
   public abstract void actionPerformed(AnActionEvent anActionEvent);
 
   protected void doRebase(AnActionEvent anActionEvent, BaseGitMacheteNonRootBranch branchToRebase) {
-    Project project = anActionEvent.getProject();
-    assert project != null : "Can't get project from anActionEvent variable";
-
-    IGitMacheteRepository macheteRepository = getPresentMacheteRepository(anActionEvent);
-
-    GitRepository gitRepository = getPresentIdeaRepository(anActionEvent);
-
-    doRebase(project, macheteRepository, gitRepository, branchToRebase);
+    Project project = ActionUtils.getProject(anActionEvent);
+    Option<IGitMacheteRepository> gitMacheteRepository = getGitMacheteRepository(anActionEvent);
+    Option<GitRepository> gitRepository = getSelectedVcsRepository(anActionEvent);
+
+    if (gitMacheteRepository.isDefined() && gitRepository.isDefined()) {
+      doRebase(project, gitMacheteRepository.get(), gitRepository.get(), branchToRebase);
+    } else {
+      LOG.warn("Skipping the action because gitMacheteRepository and/or gitRepository is empty");
```

Comment:
`empty` is misleading. It is understandable here but in log level would be ambiguous. Maybe `undefined`?

Reply:
Done, fixed everywhere